pkg/operator/rbac.go

```diff
@@ -82,12 +83,12 @@ func AssertClusterRole(clients *clients.Clients, clusterRoleName string) {
 }
 
 func AssertServiceAccountAfterDisable(clients *clients.Clients, ns, targetSA string) {
-	log.Printf("Verifying that service account %s doesn't exists\n", targetSA)
 	err := wait.Poll(config.APIRetry, config.APITimeout, func() (bool, error) {
+		log.Printf("Verifying that service account %s doesn't exists\n", targetSA)
 		saList, err := clients.KubeClient.Kube.CoreV1().ServiceAccounts(ns).List(context.TODO(), metav1.ListOptions{})
 		for _, item := range saList.Items {
 			if item.Name == targetSA {
-				return false, fmt.Errorf("found serviceaccount %s/%s", ns, targetSA)
+				return false, err
```

Comment:
I don't understand this change, can you explain please? Can the `saList` and `err` contain data at the same time?

Reply:
I will explain this in a call